config/zones.json

```diff
@@ -1772,6 +1772,9 @@
     "timezone": "Europe/Istanbul"
   },
   "TW": {
+    "capacity": {
+      "hydro_storage": 2602
```

Comment:
If I remember correctly the key is with a space instead of underscore. Can you check previous examples?